tools/java/common/src/com/google/i18n/phonenumbers/BuildMetadataFromXml.java

```diff
@@ -445,70 +465,68 @@ static boolean isValidNumberType(String numberType) {
+      metadata.voicemail =
+          processPhoneNumberDescElement(generalDesc, element, VOICEMAIL, liteBuild);
+      metadata.noInternationalDialling =
           processPhoneNumberDescElement(generalDesc, element, NO_INTERNATIONAL_DIALLING,
-          liteBuild));
-      metadata.setSameMobileAndFixedLinePattern(
-          metadata.getMobile().getNationalNumberPattern().equals(
-          metadata.getFixedLine().getNationalNumberPattern()));
+          liteBuild);
+      metadata.sameMobileAndFixedLinePattern =
+          metadata.mobile.nationalNumberPattern.equals(metadata.fixedLine.nationalNumberPattern);
     } else {
       // Set fields used only by short numbers.
-      metadata.setStandardRate(
-          processPhoneNumberDescElement(generalDesc, element, STANDARD_RATE, liteBuild));
-      metadata.setShortCode(
-          processPhoneNumberDescElement(generalDesc, element, SHORT_CODE, liteBuild));
-      metadata.setCarrierSpecific(
-          processPhoneNumberDescElement(generalDesc, element, CARRIER_SPECIFIC, liteBuild));
-      metadata.setEmergency(
-          processPhoneNumberDescElement(generalDesc, element, EMERGENCY, liteBuild));
+      metadata.standardRate =
+          processPhoneNumberDescElement(generalDesc, element, STANDARD_RATE, liteBuild);
+      metadata.shortCode =
+          processPhoneNumberDescElement(generalDesc, element, SHORT_CODE, liteBuild);
+      metadata.carrierSpecific =
+          processPhoneNumberDescElement(generalDesc, element, CARRIER_SPECIFIC, liteBuild);
+      metadata.emergency =
+          processPhoneNumberDescElement(generalDesc, element, EMERGENCY, liteBuild);
     }
 
     // Set fields used by both regular length and short numbers.
-    metadata.setTollFree(
-        processPhoneNumberDescElement(generalDesc, element, TOLL_FREE, liteBuild));
-    metadata.setPremiumRate(
-        processPhoneNumberDescElement(generalDesc, element, PREMIUM_RATE, liteBuild));
+    metadata.tollFree =
+        processPhoneNumberDescElement(generalDesc, element, TOLL_FREE, liteBuild);
+    metadata.premiumRate =
+        processPhoneNumberDescElement(generalDesc, element, PREMIUM_RATE, liteBuild);
   }
 
   // @VisibleForTesting
   static PhoneMetadata loadCountryMetadata(String regionCode, Element element, boolean liteBuild,
       boolean isShortNumberMetadata, boolean isAlternateFormatsMetadata) {
     String nationalPrefix = getNationalPrefix(element);
-    PhoneMetadata.Builder metadata =
+    PhoneMetadata metadata =
         loadTerritoryTagMetadata(regionCode, element, nationalPrefix);
     String nationalPrefixFormattingRule =
         getNationalPrefixFormattingRuleFromElement(element, nationalPrefix);
-    loadAvailableFormats(metadata, element, nationalPrefix.toString(),
-                         nationalPrefixFormattingRule.toString(),
+    loadAvailableFormats(metadata, element, nationalPrefix, nationalPrefixFormattingRule,
```

Comment:
As you can see in the lines immediately above, these are already strings, so it wasn't necessary. I think it was necessary at some point in the past, but hasn't been for a while.